src/app/zap-templates/zcl/data-model/chip/identify-cluster.xml

```diff
@@ -18,6 +18,7 @@ limitations under the License.
   <domain name="CHIP"/>
 
   <enum name="IdentifyIdentifyType" type="ENUM8">
+    <cluster code="0x0003"/>
```

Comment:
Is this temporary pending Zap changes so we can move these inside the `<cluster>` bit?